srcpkgs/spek-alternative/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'spek-alternative'
+pkgname=spek-alternative
+version=0.8.2.3
+revision=1
+build_style=gnu-configure
+hostmakedepends="automake gettext-devel intltool libtool pkg-config wxWidgets-common"
+makedepends="ffmpeg-devel wxWidgets-devel"
```

Comment:
I think should be `wxWidgets-gtk3-devel`

Reply:
I'll try building and get back to you

Reply:
It fails with this output: http://dpaste.com/3HKXJWW.txt

This is the template I used: http://dpaste.com/370Q620.txt

Reply:
Yeah, the gtk3 version uses `wx-config-gtk3`.

Reply:
I am sorry. Will try again and let you know.

Reply:
wxWidgets-gtk3-devel has the `wx-config-gtk3` binary. You might need to tell the build system to use it, if it doesn't use the `.pc` correctly. But it should be fine.

Reply:
It's defined here but I can't understand what it's doing. https://github.com/withmorten/spek-alternative/blob/0.8.2.3/configure.ac#L53

You might try opening an issue with the dev asking how to specify the wx-config binary.

Reply:
Somehow I understand. I have opened an issue.

Seems like the AM_OPTIONS_WXCONFIG is defined in `/usr/share/aclocal/wxwin.m4` in package `wxWidgets-common`. I'll look into it tomorrow more and see if I can spin something up.

Reply:
That got taken care of. I'll mark this as resolved. Thanks for helping me out.

Reply:
well that works, great!